simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -25,53 +25,53 @@
   <node
     name="squilliamson" pkg="gazebo_ros" type="spawn_model"
     args="-file $(find sub8_gazebo)/models/torpedo_board/squilliamson.sdf -sdf
-      -x 24 -y 11 -z -4.72 -Z 0 -model squilliamson"
+      -x 32.3 -y 0 -z -4.72 -Z -1.43829 -model squilliamson"
     output="screen" />
 
     <node
       name="tentacle" pkg="gazebo_ros" type="spawn_model"
       args="-file $(find sub8_gazebo)/models/tentacle/tentacle.sdf -sdf
-        -x 24 -y 10 -z -4.4 -Z 0 -model tentacle"
+        -x 34.17 -y 0 -z -4.4 -Z -.898665 -model tentacle"
       output="screen" />
 
   <!-- Attempting to spawn the bin and lid -->
   <node
     name ="bin_assembly" pkg="gazebo_ros" type="spawn_model"
     args="-file $(find sub8_gazebo)/models/bin/bin_assembly.sdf -sdf
-    -x -22.0 -y 14.0 -z -4.52 -Z -1.57 -model bin_assembly"
+    -x 32.08 -y 2.833 -z -4.52 -Z -1.57 -model bin_assembly"
     output="screen" />
 
     <node
       name ="bin_assembly2" pkg="gazebo_ros" type="spawn_model"
       args="-file $(find sub8_gazebo)/models/bin/bin_assembly2.sdf -sdf
-      -x -22.0 -y 15.25 -z -4.52 -Z -1.57 -model bin_assembly2"
+      -x 33.5 -y 3.27 -z -4.52 -Z -1.57 -model bin_assembly2"
       output="screen" />
 
     <node
       name ="lid" pkg="gazebo_ros" type="spawn_model"
       args="-file $(find sub8_gazebo)/models/bin/lid.sdf -sdf
-      -x -22.3 -y 14.16 -z -4.47 -Z -1.57 -model lid"
+      -x 31.7 -y 3.0 -z -4.47 -Z -1.57 -model lid"
       output="screen" />
 
   <!-- Attempting to spawn the start gate -->
   <node
       name ="start_gate" pkg="gazebo_ros" type="spawn_model"
       args="-file $(find sub8_gazebo)/models/start_gate/start_gate.sdf -sdf
-      -x 15 -y 10 -z -2 -Z -1.57 -model start_gate"
+      -x 3.22 -y 26.58 -z -2 -Z -1.57 -model start_gate"
```

Comment:
This new location doesn't give much room to see the gate if the sub starts behind it.

Also, can you move the first path marker to be in front of the start gate and point towards buoys? This is how the course at the transdec will be.